docs/running-headscale-linux.md

```diff
@@ -95,3 +101,107 @@ connect a node to `headscale` during the `tailscale` command:
 ```shell
 tailscale up --login-server <YOUR_HEADSCALE_URL> --authkey <YOUR_AUTH_KEY>
 ```
+
```

Comment:
We normally refer anyway to the [config-example.yaml](https://github.com/juanfont/headscale/blob/main/config-example.yaml). I don't want to maintain another example config.